tests/sessions_tests/tests.py

```diff
@@ -250,6 +250,15 @@ def test_custom_expiry_datetime(self):
         age = self.session.get_expiry_age(modification=modification)
         self.assertEqual(age, 10)
 
+        self.session.set_expiry(None)
```

Comment:
These additions seem to pass even before the fix is applied. Are they meant as regression tests or something else? Some comments to explanation the purpose could be helpful.